icons/insomnia/insomnia-original-wordmarkss.svg

```diff
@@ -0,0 +1 @@
719-2.605469 2.417969-2.605469 1.371094 0 2.203125.902343 2.203125 2.410156v6.367187h2.023438v-6.609374c0-2.574219-1.414063-3.921876-3.535157-3.921876-1.5625 0-2.582031.726563-3.0625 1.828126h-.128906v-1.691407h-1.941406v10.394531h2.023437Zm17.128906-1.683594c-.417968-1.617188-1.683593-2.675782-3.925781-2.675782-2.347656 0-4.007812 1.238282-4.007812 3.082032 0 1.472656.890625 2.457031 2.835937 2.890625l1.757813.382812c1 .222657 1.46875.671875 1.46875 1.320313 0 .804687-.859375 1.4375-2.183594 1.4375-1.210937 0-1.988281-.523438-2.230469-1.546875l-1.949218.300781c.335937 1.839844 1.859374 2.875 4.191406 2.875 2.507812 0 4.246094-1.332031 4.246094-3.214844 0-1.46875-.933594-2.375-2.839844-2.816406l-1.648438-.378906c-1.144531-.269532-1.636718-.65625-1.628906-1.359375-.007812-.796875.859375-1.367188 2.007812-1.367188 1.257813 0 1.835938.695313 2.074219 1.394531Zm6.796876 8.066406c2.925781 0 4.839843-2.144531 4.839843-5.359375 0-3.234375-1.914062-5.382813-4.839843-5.382813-2.925782 0-4.839844 2.148438-4.839844 5.382813 0 3.214844 1.914062 5.359375 4.839844 5.359375Zm.007812-1.699219c-1.914062 0-2.804688-1.671875-2.804688-3.667969 0-1.988281.890626-3.679687 2.804688-3.679687 1.898438 0 2.789062 1.691406 2.789062 3.679687 0 1.996094-.890624 3.667969-2.789062 3.667969Zm7.085938 1.488281h2.023437v-6.40625c0-1.402343.980469-2.378906 2.128906-2.378906 1.121094 0 1.898438.746094 1.898438 1.875v6.910156h2.015625v-6.625c0-1.25.761718-2.160156 2.082031-2.160156 1.066406 0 1.945313.597656 1.945313 2.003906v6.78125h2.019531v-6.96875c0-2.375-1.324219-3.5625-3.203125-3.5625-1.492188 0-2.617188.71875-3.117188 1.828126h-.105468c-.453126-1.136719-1.40625-1.828126-2.792969-1.828126-1.371094 0-2.390625.683594-2.824219 1.828126h-.128906v-1.691407h-1.941406Zm18.84375-6.171874c0-1.65625 1.015624-2.605469 2.417968-2.605469 1.375 0 2.203125.902343 2.203125 2.410156v6.367187h2.023438v-6.609374c0-2.574219-1.414063-3.921876-3.535157-3.921876-1.5625 0-2.582031.726563-3.0625 1.828126h-.128906v-1.691407h-1.9375v10.394531h2.019532Zm9.355468 6.171874h2.019532V13.957031h-2.019532Zm1.019532-12c.695312 0 1.277343-.539062 1.277343-1.203124 0-.664063-.582031-1.210938-1.277343-1.210938-.703126 0-1.277344.546875-1.277344 1.210938 0 .664062.574218 1.203124 1.277344 1.203124Zm6.738281 12.230469c1.71875 0 2.683593-.871093 3.070312-1.648437h.082031v1.417968h1.972657v-6.902343c0-3.023438-2.378907-3.628907-4.027344-3.628907-1.878906 0-3.609375.757813-4.285156 2.65625l1.898437.429688c.296875-.734375 1.054688-1.445312 2.414063-1.445312 1.304687 0 1.972656.683593 1.972656 1.859374v.046876c0 .738281-.757813.726562-2.625.941406-1.964844.230468-3.980469.746094-3.980469 3.105468 0 2.046876 1.535156 3.167969 3.507813 3.167969Zm.441406-1.625c-1.144531 0-1.96875-.511719-1.96875-1.515625 0-1.082031.960937-1.46875 2.128906-1.625.65625-.085937 2.210938-.261718 2.503907-.554687v1.339843c0 1.234376-.980469 2.355469-2.664063 2.355469Zm0 0" style="stroke:none;fill-rule:nonzero;fill:#4000bf;fill-opacity:1"/></svg>
```

Comment:
This file can be deleted now, since you replaced it with another wordmark file which looks good :)